app/components/primer/alpha/action_list.rb

```diff
@@ -71,7 +71,19 @@ def custom_element_name
       #   def with_divider(**system_arguments, &block)
       #   end
 
-      # Items. Items can be individual items or dividers. See the documentation for `#with_item` and `#with_divider` respectively for more information.
+      # @!parse
```

Comment:
Curious what this `@!parse` does

Reply:
Oh yeah! `@!parse` is a YARD directive that parses the comment as code. It's YARD's way of documenting methods that don't exist until runtime, etc.